plugins-builtin/extractcode-7z-system_provided/src/extractcode_7z/__init__.py

```diff
@@ -35,32 +35,31 @@
 
 
 class SevenzipPaths(LocationProviderPlugin):
-    
-def get_locations(self):
-
-        curr_dir = dirname(abspath(__file__))
-        distribution=platform.linux_distribution()[0]	
 	
-	# List of various major distributions consisting of flavors
+	def get_locations(self):
+	
+		distribution=platform.linux_distribution()[0]	
+	
+		# List of various major distributions consisting of flavors
 
-	debian_based_distro=['Ubuntu','Mint','debian']
+		debian_based_distro=['Ubuntu','Mint','debian']
 	
-	rpm_based_distro=['Fedora','redhat']
+		rpm_based_distro=['Fedora','redhat']
 
-	if distribution in debian_based_distro:
+		if distribution in debian_based_distro:
 		
-		lib_dir = '/usr/lib/p7zip'
+			lib_dir = '/usr/lib/p7zip'
 	
-	elif distribution in rpm_based_distro:
-		
-		lib_dir = '/usr/libexec/p7zip'
+		elif distribution in rpm_based_distro:
```

Comment:
Culd you make sure that you use 4 spaces throughout and no tabs?

Reply:
Will take care of that!
What's reason behind this are my tabs long spaced(8 spaces) or something else?

Reply:
The reason is simply that we never use tabs